srcmedia/scss/search/_forms.scss

```diff
@@ -0,0 +1,134 @@
+        align-items: center;
+        text-align: center;
+    }
+
+    .total-results {
+        font-family: $serif;
+        font-style: italic;
+        font-size: 0.8rem;
+        margin: 3rem 0 1rem;
+    }
+
+    input[name=query] {
+        flex: auto;
+        padding: .2rem 0;
+        font-size: 0.8rem;
+        background-color: transparent;
+        border-width: 0 0 2px 0;
+        border-color: $black;
+        border-bottom-style: dotted;
+        outline: none;
+        transition: border 0.5s;
+
+        &::placeholder {
+          color: $nearly-black;
+          opacity: 1; /* Firefox */
+        }
+
+        + label {
+            display: inline-block;
+            width: 1rem;
+            height: 1rem;
+            background-image: url('/static/img/icons/search.svg');
+            background-position: center center;
+            background-repeat: no-repeat;
+        }
+
+        &:focus {
+            border-bottom-style: solid;
+        }
+    }
+
+    fieldset {
+        width: 100%;
+        font-size: 0.8rem;
+
+        legend {
+            font-weight: bold;
+            padding: 1.75rem 0rem 0.5rem;
+        }
+    
+        // chrome doesn't allow fieldsets to use flex, so we have to do this
+        // https://bugs.chromium.org/p/chromium/issues/detail?id=262679
+        .inner { 
+            width: 100%;
+            display: flex;
+            flex-flow: row wrap;
+            justify-content: flex-start;
+            align-items: center;
+        }
+
+        label {
+            flex: auto;
+            height: 2.2rem;
+            background: white;
+            margin-right: .1rem;
+            display: flex;
+            flex-flow: row nowrap;
+            justify-content: flex-start;
+            padding: 0 .5rem;
+            align-items: center;
+            border: 2px solid white;
+            box-sizing: border-box;
+
+            &:last-child {
+                margin-right: 0;
+            }
+
+            &:hover {
+                border-color: $green;
+            }
+
+            img.icon {
+                max-height: 1.5rem;
+                margin-left: 0.5rem;
+            }
+
+            .count {
+                font-size: 0.6rem;
+            }
+
+            span[role=tooltip] {
+                position: absolute;
+                right: .5rem;
+                top: calc(50% - (.75rem / 2));
+            }
+        }
+
+        input:focus + label {
+            border-color: $green;
+        }
+
+        input[type=checkbox] + label {
+            cursor: pointer;
+        }
+
+        // uses after so it can stack with tooltip info icon
+        input[type=checkbox] + label::after {
+            content: '';
+            opacity: 0;
+            position: absolute;
+            width: .75rem;
+            height: .75rem;
+            right: .5rem;
+            top: calc(50% - (.75rem / 2));
+            background-image: url("/static/img/icons/filter_clicked_state.svg");
```

Comment:
Should we put the static img paths in a variable so it's all set in one place, in case it ever changes?

Reply:
if the image ever gets used in more than one place, totally. I can see that potentially happening for this check mark, but with the case of the little triangle for the tooltip, I'm fine leaving it as a path since it should only ever be used for a tooltip.